tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -70,7 +70,11 @@ class CloudGov(Common):
     #
     env_based_db_name = f'tdp_db_{cloudgov_space_suffix}_{cloudgov_name}'
 
-    db_name = database_creds['db_name'] if (cloudgov_space_suffix in ["prod",  "staging"]) else env_based_db_name
+    logger.debug("css: " + cloudgov_space_suffix)
+    if (cloudgov_space_suffix in ["prod",  "staging"]):
```

Comment:
It looks like there is an extra space here between prod and staging. Other than that, this all looks good to me.

Reply:
@andrew-jameson I made the change and pushed this up. just pull this branch before making other changes